layouts/partials/header-menu.html

```diff
@@ -1,4 +1,27 @@
 <ul>
-    <li><a href="{{ "about" | relURL }}">{{ i18n "about" }}</a></li>
-    <li><a href="{{ "/" | relURL }}">{{ i18n "posts" }}</a></li>
+    {{ $currentPage := . }}
+    {{ range .Site.Menus.main }}
+        {{ if .HasChildren }}
+            <li class="{{ if $currentPage.HasMenuCurrent "main" . }}active{{ end }}">
+                <a href="#">
+                    {{ .Pre }}
+                    <span>{{ .Name }}</span>
+                </a>
+            </li>
+            <ul class="sub-menu">
+                {{ range .Children }}
+                    <li class="{{ if $currentPage.IsMenuCurrent "main" . }}active{{ end }}">
+                        <a href="{{ .URL }}">{{ .Name }}</a>
+                    </li>
+                {{ end }}
+            </ul>
+        {{ else }}
+            <li>
+                <a href="{{ .URL }}">
+                    {{ .Pre }}
```

Comment:
If we include `.Pre` we should probably include `.Post` as well. See the [list of menu entry variables](https://gohugo.io/variables/menus/#menu-entry-variables) (mainly leaving this here as future reference for myself 🙂).

Reply:
Added `.Post`.